### PR TITLE
disable concurrent builds in reposync

### PR DESCRIPTION
**Pull request by adammhaile**
@sosiouxme @tbielawa

### CODE REVIEW

**Comment by adammhaile** [comment]
Also adding 4.1 to schedule per Clayton's request

File: scheduled-jobs/build/reposync/Jenkinsfile [hunk 1]
```diff
@@ -13,6 +13,14 @@ b = build       job: '../aos-cd-builds/build%2Freposync', propagate: false,
...
+
+b = build       job: '../aos-cd-builds/build%2Freposync', propagate: false,
+                parameters: [[$class: 'StringParameterValue', name: 'SYNC_VERSION', value: '4.1']]
+
+description += "${b.displayName} - ${b.result}\n"
+failed |= (b.result != "SUCCESS")
 
 currentBuild.description = description.trim()
-currentBuild.result = b.result
+currentBuild.result = failed?"FAILURE":"SUCCESS"
```
Comment (sosiouxme) [inline scheduled-jobs/build/reposync/Jenkinsfile]: Call me crazy, I like some space between my ternary args...

Comment (adammhaile) [reply scheduled-jobs/build/reposync/Jenkinsfile]: Look... I copy pasted that from another job :P Blame who wrote that.
But I'll change it.

**Pull request merged**